perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -273,12 +273,19 @@ sub open311_pre_send {
     if (my $comment_id = $row->get_extra_metadata('echo_report_reopened_with_comment')) {
         my $comment = FixMyStreet::DB->resultset('Comment')->find($comment_id);
         if ($comment && $comment->text) {
-            my $text = "Closed report has a new comment: " . $comment->text . "\n\n" . $row->detail;
+            my $text = _munge_text_for_new_investigation_event($comment, $row->detail);
             $row->detail($text);
         }
     }
 }
 
+sub _munge_text_for_new_investigation_event {
+    my ($comment, $detail) = @_;
+
+    my $text = $comment->text . '|' . $comment->user->name . ' ' . $comment->user->email . '|' . $detail;
+    return Utils::truncate_text($text, 250, '...');
```

Comment:
Is there a reason to truncate the text here? It feels like we shouldn't truncate here, and if any truncation is needed it would happen at the open311-adapter end that's formatting for sending.

Reply:
No - I guess you will end up with truncated text which encourages you to visit the report, only to find the exact same truncated text.